requirements-dev.in

```
@@ -0,0 +1,9 @@
+-c requirements.txt
```

Why not using
```suggestion
-r requirements.txt
```

And thus, you only have to install with `pip install -r requirements-dev.txt`, instead of having to install `requirements` and `requirements-{dev,prod}` separately ? This also make upgrades slightly easier, so when dependabot has upgraded something, after pulling you only have to run `pip install --upgrade -r requirements-dev.txt` on your laptop

---

I just followed the documentation 🤷 
https://github.com/jazzband/pip-tools#workflow-for-layered-requirements
I don't really have a strong opinion on how to do it :)

Btw, no need for `--upgrade` in `pip install --upgrade -r requirements-dev.txt` as the requirements are strictly pinned